pkg/tests/schema_pulumi_test.go

```diff
@@ -3979,130 +3956,6 @@ outputs:
 	assert.Equal(t, "", out.Outputs["emptyValue"].Value)
 }
 
-func TestUnknownSetElementDiff(t *testing.T) {
```

Comment:
Is this removed test covered else-where?

Reply:
yeah, the tests added in this PR cover all variations of this test